### PR TITLE
update oncoprint Ns and figure legends

### PR DESCRIPTION
**Pull request by jharenza**
### Purpose

_Briefly describe your changes here._

Previously, we left an HTML tag as a reminder to update oncoprint Ns once the tables were generated via script (as noted in #159) for ease of review/updating. Here, I updated those Ns that were missing, included some brief text about the supplemental other oncoprint which was previously missing, updated the main Fig 2 legend, and added the supplemental S3 A figure legend. 

### Issue

#159 

#### Which areas should receive a particularly close look?
double check N updates

#### Is there anything that you want to discuss further?
no

#### Is the pull request ready for review?
yes

### Pull review checklist

Unless otherwise noted above, this PR will be considered ready for review when all four items have been checked.

- [ ] All changes to text follow "one sentence per line" [[Manubot instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#manubot-markdown)]
- [ ] All citations follow the [Manubot citation instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#citations)
- [ ] All changes or additions to tables follow the [Manubot table instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#tables)
- [ ] All figures follow the [Manubot figure instructions](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#figures)
- [ ] There are no new spelling errors identified by the [Manubot spellchecker](https://github.com/AlexsLemonade/OpenPBTA-manuscript/blob/master/USAGE.md#spellchecking)

### Spellcheck Step

The dictionary used for spellchecking can be updated.
Edit the file in `build/assets/custom-dictionary.txt` by adding new entries to the end.
You do not need to change anything else.
However, if you want to update the first line to have an accurate count of words and you want to remove non-unique ones, run the following command from within `build/assets` on your favorite OS X or Linux machine:
```
(( len = $(awk '!a[$0]++' < custom-dictionary.txt | wc -l ) - 1 )); tmpfile="$(mktemp)"; echo "personal_ws-1.1 en $len utf-8" > $tmpfile; tail -n +2 custom-dictionary.txt | awk '!a[$0]++' >> $tmpfile; mv $tmpfile custom-dictionary.txt
```

### CODE REVIEW

**Comment by jharenza** [comment]
> Hi @jharenza , I need a bit more guidance here. I'm looking through the history of this PR as well as the files in `oncoprint-landscape` and I'm not seeing exactly where these numbers are coming from. As an example, this sentence -

Sorry, I meant to go back and add this info to the PR comments.

- For all Ns, I obtained them from the tables generated in https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/oncoprint-landscape/tables/
- Since we are using primary samples only, only the below tables are relevant:
```
harenzaj@38f9d38f36c9 tables % ls -lh primary_only* 
-rw-r--r--  1 harenzaj  1768498755   2.3K Oct 28 13:59 primary_only_embryonal_oncoprint_summary_n.tsv
-rw-r--r--  1 harenzaj  1768498755   4.2K Oct 28 13:59 primary_only_hgat_oncoprint_summary_n.tsv
-rw-r--r--@ 1 harenzaj  1768498755   1.0K Oct 28 13:59 primary_only_lgat_oncoprint_summary_n.tsv
-rw-r--r--  1 harenzaj  1768498755   1.2K Oct 28 13:59 primary_only_n_per_cancer_group.tsv
-rw-r--r--  1 harenzaj  1768498755   1.8K Oct 28 13:59 primary_only_other_oncoprint_summary_n.tsv
```
- I utilized the `AlteredSamples` column as the numerator for genes of interest per histology being discussed. If using the broad histology, those were obtained from `*embryonal*`, `*hgat*`, `*lgat*` ,or `*other*`. For genes of interest within a specific cancer group, it was generally small enough for me to count off of the oncoprint (I know, 😬), and then I used the denominator from  the cancer group, found in `primary_only_n_per_cancer_group.tsv`.

For you example, 

>> > We observed 27% (16/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 77% (23/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_.
> 
> I see how we have 60 EPN in this [primary only file](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/oncoprint-landscape/tables/primary_only_n_per_cancer_group.tsv), but I can't tell where `16` is from...I'm guessing (?) this was calculated from information in the `fusion-summary` module?

This one was a combination of visually inspecting `primary_only_other_oncoprint.pdf` (only EPN samples have this fusion and gathering the N from `primary_only_other_oncoprint_summary_n.tsv`. I did not have to use the fusion module at all.

> It's also not clear to me where the `23` numerator for craniopharyngiomas comes from at all. I see 23 altered samples for CTNNB1 in table `tables/primary_only_other_oncoprint_summary_n.tsv`, but no indication this 23 is specifically craniopharyngiomas.

This is also the same situation as above - only CRANIO samples in the oncoprint have this alteration (expected), so therefore, I used the N from the table as only cranios.

Please let me know if this helps or if there are other not straightforward examples that I can explain!

**Comment by jaclyn-taroni** [comment]
@jharenza we should be able to point to a place where we get these numbers in the analysis repository. We can add a notebook to the end of the oncoprint module that counts alterations within whatever groups are being presented or update the script where the tables are being generated to include what you will report in the text.

**Comment by sjspielman** [comment]
@jaclyn-taroni having a notebook for this makes sense to me, and then once the notebook is approved we can integrate those numbers into the MS. I'll go ahead and open an issue about this in the analysis repo.

**Comment by jharenza** [comment]
@sjspielman I have just updated this based on https://github.com/AlexsLemonade/OpenPBTA-analysis/pull/1266, so it is ready for review after 1266 goes in!

File: content/03.results.md [hunk 1]
```diff
@@ -100,7 +100,6 @@ We  performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and f
 Following SNV consensus calling (**Figure S1** and **Figure S2, panels A-F**), we observed lower expected tumor mutation burden (TMB) **Figure S2, panel G** in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors only found within HGATs.
```
Comment (sjspielman) [inline content/03.results.md:100]: Not strictly related to PR but why not. Should we add in the Campbell 2017 reference here for defining hypermutant and ultrahypermutant?  The DOI is `10.1016/j.cell.2017.09.048`

File: content/03.results.md [hunk 2]
```diff
@@ -120,13 +119,13 @@ The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb)
 
 #### Other CNS tumors
 **Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors (N = 195). 
-We observed 8% (16/195) of tumors to be _C11orf95::RELA_ fusion positive ependymomas and 12% (23/195) to be adamantinomatous craniopharyngiomas driven by mutations in _CTNNB1_. 
-<!-- TODO: update NF2 cancer types -->
-Multiple cancer types contained somatic mutations or fusions in _NF2_ (7%, comprised of ependymomas, desmoplastic infantile astrocytoma and ganglioglioma, …, and …) and _BRAF_ (4%), in addition to rare ependymoma fusions (< 4% of all embryonal tumors) in _YAP1_, _KRAS_, _ERBB4_, and _MYB_. 
-The majority of dysembryoplastic tumors harbored alterations in _FGFR1_, including _FGFR1::TACC1_ fusions.
-<!-- TODO: update numbers/ add info for supplemental oncoprint -->
+We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/03.results.md:122]: Also noting N=60 ependymoma  and N=30 craniopharyngiomas ✅ 
https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/oncoprint-landscape/tables/primary_only_n_per_cancer_group.tsv

File: content/03.results.md [hunk 3]
```diff
@@ -120,13 +119,13 @@ The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb)
 
 #### Other CNS tumors
 **Figure {@fig:Fig2}D** depicts an oncoprint for the remaining primary CNS tumors (N = 195). 
-We observed 8% (16/195) of tumors to be _C11orf95::RELA_ fusion positive ependymomas and 12% (23/195) to be adamantinomatous craniopharyngiomas driven by mutations in _CTNNB1_. 
-<!-- TODO: update NF2 cancer types -->
-Multiple cancer types contained somatic mutations or fusions in _NF2_ (7%, comprised of ependymomas, desmoplastic infantile astrocytoma and ganglioglioma, …, and …) and _BRAF_ (4%), in addition to rare ependymoma fusions (< 4% of all embryonal tumors) in _YAP1_, _KRAS_, _ERBB4_, and _MYB_. 
-The majority of dysembryoplastic tumors harbored alterations in _FGFR1_, including _FGFR1::TACC1_ fusions.
-<!-- TODO: update numbers/ add info for supplemental oncoprint -->
+We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
+Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas.
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/03.results.md:123]: Also noting N=11 schwannomas ✅ 
N=7 meningiomas ✅

File: content/03.results.md [hunk 4]
```diff
@@ -120,13 +119,13 @@ The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb)
...
-<!-- TODO: update NF2 cancer types -->
-Multiple cancer types contained somatic mutations or fusions in _NF2_ (7%, comprised of ependymomas, desmoplastic infantile astrocytoma and ganglioglioma, …, and …) and _BRAF_ (4%), in addition to rare ependymoma fusions (< 4% of all embryonal tumors) in _YAP1_, _KRAS_, _ERBB4_, and _MYB_. 
-The majority of dysembryoplastic tumors harbored alterations in _FGFR1_, including _FGFR1::TACC1_ fusions.
-<!-- TODO: update numbers/ add info for supplemental oncoprint -->
+We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
+Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas. 
+Rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ were observed in 12% (7/60) of ependymoma tumors. 
+DNETs harbored alterations in MAPK/PI3K pathway genes as previously reported [@doi:10.1093/jnen/nlz101], including _FGFR1_ (21%, 4/19), _PDGFRA_ (10%, 2/19), and _BRAF_ (5%, 1/19).
```
[the remark this answers is not in the document]
Comment (sjspielman) [reply content/03.results.md:125]: N=20 DNET ✅

File: content/03.results.md [hunk 5]
```diff
@@ -120,13 +119,13 @@ The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb)
...
...ioma (N = 35), subependymal giant cell astrocytoma (N = 2), diffuse fibrillary astrocytoma (N = 1), pilocytic astrocytoma (N = 1), and pleomorphic xanthoastrocytoma (N = 1); B, Embryonal tumors (N = 128): medulloblastomas (N = 95), atypical teratoid rhabdoid tumors (N = 24), embryonal tumors with multilayer rosettes (N = 2), other CNS embryonal tumors (N = 5), ganglioneuroblastoma (N = 1), and CNS neuroblastoma (N = 1); C, Diffuse astrocytic and oligodendroglial tumors (N = 61): diffuse midline gliomas (N = 34) and non-midline high-grade gliomas (N = 26), oligodendroglioma (N = 1); D, Other CNS tumors (N = 195): ependymomas (N = 60), dysembryoplastic neuroepithelial tumors (N = 19), meningiomas (N = 17), schwannoma (N = 11), neurofibroma plexiform (N = 7). Other CNS tumors with N < 5 are displayed in **Figure {@fig:FigS3}A**. Patient sex (`germline_sex_estimate`) and tumor histology (`cancer_group`) are displayed as annotations at the bottom of each plot. Only samples with mutations in the listed genes are shown. Multiple CNVs are denoted as a complex event.](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/figures/pngs/primary_only_oncoprint_landscape.png){#fig:Fig2}
```
Comment (sjspielman) [inline content/03.results.md:128]: Consistent with https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/oncoprint-landscape/tables/primary_only_n_per_cancer_group.tsv

File: content/07.supplemental.md [hunk 6]
```diff
@@ -15,6 +15,13 @@ Upset plot (F) showing overlap of variant calls.
 Cumulative distribution TMB plots for PBTA (G) and TCGA (H)tumors using consensus SNV calls.
 
 **Figure S3. Genomic instability of pediatric brain tumors, Related to Figures 2 and 3.**
+(A) Oncoprint of canonical somatic gene mutations, CNVs, fusions, and TMB (top bar plot) for the top 20 genes mutated across rare CNS tumors (N < 5 each): desmoplastic infantile astrocytoma and ganglioglioma (N = 1), germinoma (N = 4), glial-neuronal NOS (N = 4), metastatic secondary tumors (N = 3), neurocytoma (N = 2), and pineoblastoma (N = 3).
```
Comment (sjspielman) [inline content/07.supplemental.md:18]: Consistent with https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/analyses/oncoprint-landscape/tables/primary_only_n_per_cancer_group.tsv

**Review by sjspielman — APPROVED** [review]
🚀

File: content/03.results.md (hunk 1 above)
Comment (jharenza) [reply content/03.results.md:100]: yes, good suggestion

File: content/03.results.md [hunk 7]
```diff
@@ -120,13 +119,13 @@ The two tumors with ultra-high tumor mutation burden (TMB) (> 100 Mutations/Mb)
...
-We observed 8% (16/195) of tumors to be _C11orf95::RELA_ fusion positive ependymomas and 12% (23/195) to be adamantinomatous craniopharyngiomas driven by mutations in _CTNNB1_. 
-<!-- TODO: update NF2 cancer types -->
-Multiple cancer types contained somatic mutations or fusions in _NF2_ (7%, comprised of ependymomas, desmoplastic infantile astrocytoma and ganglioglioma, …, and …) and _BRAF_ (4%), in addition to rare ependymoma fusions (< 4% of all embryonal tumors) in _YAP1_, _KRAS_, _ERBB4_, and _MYB_. 
-The majority of dysembryoplastic tumors harbored alterations in _FGFR1_, including _FGFR1::TACC1_ fusions.
-<!-- TODO: update numbers/ add info for supplemental oncoprint -->
+We observed 25% (15/60) of ependymoma tumors to be _C11orf95::RELA_ (now, _ZFTA::RELA_) fusion positive ependymomas and 70% (21/30) of craniopharyngiomas to be driven by mutations in _CTNNB1_. 
+Multiple histologies contained somatic mutations or fusions in _NF2_: 41% (7/17) of meningiomas, 5% (3/60) of ependymomas,  and 27% (3/11) schwannomas. 
+Rare fusions in _ERBB4_, _YAP1_, _KRAS_, and _MAML2_ were observed in 12% (7/60) of ependymoma tumors.
```
[the remark this answers is not in the document]
Comment (jharenza) [reply content/03.results.md:124]: you're right, thanks!

**Pull request merged**